prophyc/generators/schema.py

```diff
@@ -0,0 +1,134 @@
+from collections import namedtuple
+
```

Comment:
There is some whitespace trouble in generated output -> the generated file starts with newline and contains double newline between structures.
```
(py36) aurzenligl@aurzenligl-pc /tmp/1 $ cat x.prophy 
struct X {
    u32 x;
    u64 y[2];
};

struct Y {
    u16 q;
};
(py36) aurzenligl@aurzenligl-pc /tmp/1 $ cat out/x.prophy

struct X {
    u32 x;
    u64 y[2];
};


struct Y {
    u16 q;
};
```

Reply:
Oh, yes, It's because (only) structures get extra newlines around in contrast to any other model kinds. ...and the rendering mechanism is not clever enough to make it clean in each case. If there would be other kinds around - you would probably miss the space between them. Of course, I'll try to fix it.

Reply:
I managed to manage the whitespace but not perfectly. I added also a test case that you raised here. It's in `prophyc/tests/generators/test_prophy_gen.py :: test_structs` currently it outputs no newlines around and two in-between:
```
struct X {
    u32 x;
    u64 y[2];
};


struct Y {
    u16 q;
};
```